lib/RateLimiterMongo.js

```diff
@@ -235,7 +257,7 @@ class RateLimiterMongo extends RateLimiterStoreAbstract {
     const where = Object.assign({ key: rlKey }, docAttrs);
 
     return this._collection.deleteOne(where)
-      .then(res => res.result.n > 0);
+      .then(res => res.deletedCount > 0);
```

Comment:
I discovered this while testing with the real clients: v4 does not return `result` anymore. Seems to be in line with https://docs.mongodb.com/manual/reference/method/db.collection.deleteOne/

Reply:
> I discovered this while testing with the real clients: v4 does not return `result` anymore. Seems to be in line with https://docs.mongodb.com/manual/reference/method/db.collection.deleteOne/

Should it be selected depending on driver version?

Reply:
It worked fine with 3.6.9, but let me do some digging.

Reply:
It looks like this was updated in https://github.com/mongodb/node-mongodb-native/pull/2651/files#diff-06ad5e5a8b0f21c69f986da5545842d224dd5a79c054081bc35d714e4bdc6077. However, it just mentions that `Our CRUD result types have been out-of-sync with the official drivers CRUD specification for some time`. I could not find any old specs where `result` was part of the response.

Reply:
@backflip `deletedCount` is supported at least from MongoDB Server 3.2 http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#~deleteWriteOpResult It is safe to use it

Reply:
@backflip Did it brake tests, where `delete` function used? I see 3 mocks with `result.n` mocked for those tests.

Reply:
@backflip never mind above question. I see you already took care of that 👍